src/_P036_FrameOLED.ino

```diff
@@ -316,7 +365,7 @@ boolean Plugin_036(byte function, struct EventStruct *event, String& string)
 
         for (byte varNr = 0; varNr < P36_Nlines; varNr++)
         {
-          addFormTextBox(String(F("Line ")) + (varNr + 1), getPluginCustomArgName(varNr), P036_displayLines[varNr], P36_Nchars);
+          addFormTextBox(String(F("Line ")) + (varNr + 1), getPluginCustomArgName(varNr), P036_DisplayLinesV1[varNr].Content, P36_NcharsV1-1);
```

Comment:
No check for the version here?  (Same applies to a number of other places in the code)

Reply:
No need, always saving the CustomTaskSetting in the new V1 format!